cdap-app-templates/cdap-etl/cdap-data-pipeline2_2.11/pom.xml

```diff
@@ -97,14 +97,6 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-core_2.11</artifactId>
       <scope>provided</scope>
-      <exclusions>
```

Comment:
why is this no longer excluded?

Reply:
It's provided scope, so it's not included anyway.
Original code was 
```
<dependency>
      <groupId>co.cask.cdap</groupId>
      <artifactId>cdap-etl-batch</artifactId>
      <version>${project.version}</version>
      <exclusions>
        <exclusion>
          <groupId>co.cask.cdap</groupId>
          <artifactId>hydrator-spark-core</artifactId>
        </exclusion>
        <exclusion>
          <!-- CDAP-12742: avoid packaging commons-lang3, which is a dependency of cdap-formats (from grok),
                           but clashes with the one from Spark -->
          <groupId>org.apache.commons</groupId>
          <artifactId>commons-lang3</artifactId>
        </exclusion>
      </exclusions>
    </dependency>```
Now with "provided" this exclusion has no meaning